dask_sql/physical/rex/core/call.py

```diff
@@ -613,6 +614,37 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        # TODO: format timestamps for GPU tests
+        if "cudf" in str(type(df)):
+            if format != "%Y-%m-%d %H:%M:%S":
+                print("Formatting timestamps not supported on GPU")
+            if df.dtype == "object":
+                return df
+            else:
+                return df * 10**9
+        # String cases
+        elif type(df) == str:
+            return datetime.strptime(df, format)
+        elif df.dtype == "object":
+            df = dd.to_datetime(df)
+        # Integer cases
+        elif np.isscalar(df):
+            df = datetime.utcfromtimestamp(df)
+            return df.strftime(format)
+        else:
+            df = dd.to_datetime(df, unit="s")
+        return df.dt.strftime(format)
```

Comment:
The `to_timestamp` method is expected to return a datetime64 object rather than a string. The `format` argument here specified how the input may be formatted for strings but doesn't require the output to be formatted that way.

Reply:
Oh, I interpreted `format` to be the format of the timestamp that we want to be returned, i.e., setting `format="yyyy-mm-dd"` is telling Dask-SQL that we want the outputted column to be a datetime64 object to be formatted as `yyyy-mm-dd`. From the PostgreSQL docs, it seems more obvious that `format` is supposed to be telling Dask-SQL what format the user entered the timestamp as. I can update that.

I was also looking at PySpark docs when writing the formatting patterns: https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.functions.to_timestamp.html